defs.bzl

```diff
@@ -23,9 +23,14 @@ load("//internal:ts_config.bzl", _ts_config = "ts_config")
 load("//internal/devserver:ts_devserver.bzl", _ts_devserver = "ts_devserver_macro")
 load(
     "//internal/karma:ts_web_test.bzl",
-    _ts_web_test = "ts_web_test_macro",
+    _ts_web_test = "ts_web_test",
     _ts_web_test_suite = "ts_web_test_suite",
 )
+load(
+    "//internal/karma:karma_web_test.bzl",
```

Comment:
not sure we should put this in the public API yet.

I think generally we should have some process for maturing new rules before they go in the public API